### ISSUE TITLE
Improve boost dB jumps

### ISSUE THREAD

**Issue by elieserdejesus** [elieserdejesus/JamTaba#917]
@Hitrate proposed: 
> Another point about the buttons: the -12 to +12 db burton is very obtrusive and abrupt in jumping this 24(!!) db’s !

> Suggestion: Change it to -6db, -12db jump back to 0db, then jump to +6db, then +12db
Often 12 db is way too much, and by doing t that way you’d retain the -12 and +12 option but add -6 and +6 and avoid ruined ears by making too big a jump audio decibel wise :)

Now JamTaba is using 2 boost buttons, is not complicated change the jump steps.

**Important remember**: is necessary consider the old boost values stored in users machines. First idea is use a new **UsersDataCache** version and ignore boost values from old UsersDataCache.

**Comment by jonjamcam** [comment]
>  First idea is use a new UsersDataCache version and ignore boost values from old UsersDataCache.

yeah. I agree there should be no problem to "reset" the button's previous setting so the new button can store a wider range of options.
